### DISCUSSION TITLE
dark mode

### DISCUSSION THREAD

**Discussion by Disablio** [ciderapp/Apple-Music-Electron#167] [Q&A]
Hi im not really good at doing stuff like this and it's probably a basic question  but I was wondering how to add themes like dark mode and such because I just have the default light mode. Also my pc is set to dark mode but it doesnt change. Thanks.

**Comment by coredev-uk** [comment]
If the application is not by default dark mode, if you go to Preferences (click on your profile picture and in the dropdown press preferences or do CTRL + ALT + S), go to advanced and under force application mode set it to forcedarkmode.

**Reply by Disablio** [reply]
sorry im bad at understanding. what do I click on my picture on? because on apple music eleectron nothing happened

**Reply by coredev-uk** [reply]
![img](https://i.imgur.com/vDXb1cf.png)

Click on the profile picture and this will show a dropdown

![image](https://i.imgur.com/cu2OmWQ.png)

Click on preferences and this will show a window 

![img](https://i.imgur.com/iP9mWxB.png)

Then click advanced settings and under **forceApplicationMode** there is a drop-down, select forceDarkMode then a prompt will ask you to restart then go ahead and press restart now
